examples/contracts/aggregator-consumer/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Why does this file need to be any different than this file: https://github.com/smartcontractkit/chainlink-starknet/blob/develop/packages-ts/starknet-gauntlet/tsconfig.json

Reply:
Looks like you've removed this file, while it should be part of the project.

Please c/p the one from the above link, and add it to the `aggregator-consumer` project.